.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
+branch = true
+source =
+    suitcase.csv
+omit = suitcase/csv/_version.py
+
+[report]
+exclude_lines =
+    raise NotImplemented
```

Comment:
I haven't seen this done on other projects. Just curious, what's the rationale for not testing `__str__` or `__repr__`?